### ISSUE TITLE
Error: ENOENT: no such file or directory, scandir '/var/task/user/resources

### ISSUE THREAD

**Issue by RustyDev** [MONEI/Shopify-api-node#244]
```bash
Error while initializing entrypoint:
{ Error: ENOENT: no such file or directory, scandir '/var/task/user/resources'
```

I'm seeing this error when trying to use this package with Now v2:

https://shopify-error-qnptwbcf5.now.sh/_logs
https://shopify-error-qnptwbcf5.now.sh/_src

```javascript
const Shopify = require('shopify-api-node');

module.exports = (req, res) => {
  res.end(`Hello from Node.js on Now 2.0!`);
};
```

It's the only package that I'm importing into my project and when I remove it, the deploy works as expected. Any idea where that's coming from?

**Comment by hhff** [comment]
hi @RustyDev - I'm getting this currently. Do you recall how you fixed this?

Thank you!

**Comment by hhff** [comment]
For anyone else who hits this, we have a fork over here that hard codes the resources:
https://github.com/sanctuarycomputer/Shopify-api-node

**Comment by parkerproject** [comment]
@hhff that didn't fix it for me.

**Comment by diogoviannaaraujo** [comment]
Hi, having this problem right now, does anyone knows when resource folder is wiped?

**Comment by diogoviannaaraujo** [comment]
@hhff Im still getting the error with your version. I guess this should be reopen.

**Comment by parkerproject** [comment]
@diogoviannaaraujo nope, still having the issue. I just decided to make requests to Shopify API endpoints directly without using this library. You might find this helpful [example using axios](https://community.shopify.com/c/Shopify-APIs-SDKs/Shopify-GraphQL-and-Node-library/m-p/510262#M33140)

**Comment by tim-phillips** [comment]
I've been using @hhff fork for a while with no problems.

#296 fixes it in this library, try `v2.22.0`
